### DISCUSSION TITLE
Create RabbitMQ Streams - CLI commands

### DISCUSSION THREAD

**Discussion by praveengopalan** [rabbitmq/rabbitmq-server#10616] [Questions] [answered]
Hi Team, 
I am new to rabbitmq streams. In our qa/uat environment, the code doesn't have permission to create/delete streams. So is there any way to create the streams by executing commands?

**Comment by michaelklishin** [comment, marked as answer]
Applications declare streams, not operators.

Even tools like the K8S [Topology Operator](https://www.rabbitmq.com/docs/kubernetes/operator/using-topology-operator) use client libraries under the hood, not a "special API" of sorts. All client libraries, including the HTTP API ones, eventually run into the same set of permission checks.

If they don't have sufficient permissions, they should be given them. They can use a dedicated virtual host in test environments where they would have a complete set of permissions (although that sort of defeats the purpose of testing, but you have to find a convenience/delta with production balance).

**Comment by praveengopalan** [comment]
@michaelklishin thanks for your quick response. 
could you please explain how I can update the existing super-stream by adding new partitions?
